app/models/os_major_version.rb

```diff
@@ -25,16 +25,20 @@ def self.aggregated_cast(column)
       Arel::Nodes::NamedFunction.new(
         'REPLACE',
         [
-          Xccdf::Benchmark.arel_table[:ref_id],
+          arel_table[:ref_id],
           Arel::Nodes::Quoted.new("#{Xccdf::Benchmark::REF_PREFIX}-"),
           Arel::Nodes::Quoted.new('')
         ]
       ).as('int')
     ]
   ).as('os_major_version')
 
-  PROFILE_LAST_ID = Arel.sql("(#{OsMajorVersion.aggregated_cast(Profile.arel_table[:id]).to_sql})[1] as \"id\"")
-  PROFILE_BM_VERSIONS = OsMajorVersion.aggregated_cast(Xccdf::Benchmark.arel_table[:version]).as('bm_versions')
+  PROFILE_IN_USE = Arel::Nodes::True.new.as('in_use')
+  PROFILE_LAST_ID = Arel.sql("(#{aggregated_cast(Profile.arel_table[:id]).to_sql})[1] as \"id\"")
+  PROFILE_BM_VERSIONS = aggregated_cast(arel_table[:version]).as('bm_versions')
```

Comment:
Refactoring, pls separate (all of the above).

Reply:
What do you mean?

Reply:
Are there any changes here, because I only see changes like `OsMajorVersion.aggregated_cast` -> `aggregated_cast`

Reply:
Oh, you mean separate commit?

Reply:
Yes please.